clean.cmd

```diff
@@ -0,0 +1,177 @@
+@if "%_echo%" neq "on" echo off
```

Comment:
We definitely should clean this up more but some parts, in particular the tools dir, have to be treated specially. I would like to see this script eventually come to only support one option which would be `all` and for anything else it just calls run with the other arguments.